src/pages/Proposal.vue

```diff
@@ -181,16 +189,10 @@ export default defineComponent({
 				} = values;
 				if (typeof this.daoStore.dao_id === "undefined") {
```

Comment:
We duplicated code when check error in Proposal and Vote
https://github.com/scale-it/sigmadao-ui/blob/ac184aaaeb2fb55e7b8489413405fcc7104eb629/src/pages/DepositToken.vue#L77

Reply:
yes, error conditions are repeated at multiple places, will create a common function and return error from there